aten/src/ATen/native/Copy.cpp

```diff
@@ -287,6 +288,8 @@ static Tensor & copy_impl(Tensor & self, const Tensor & src, bool non_blocking)
     device_type = kHIP;
   } else if (iter.device_type(1) == kMPS) {
     device_type = kMPS;
+  } else if (iter.device_type(1) == kXPU){
```

Comment:
hi @albanD  We may need keep this device check as this code is intended to use accelerator kernel once one of tensor(src/dst) is on accelerator device. So I think we may need XPU logic here also as we should have same logic like cuda/mps. If my understanding here is wrong, could you please correct me? Thanks.

Reply:
FYI 

https://github.com/pytorch/pytorch/pull/130088/files#diff-5920abc01985a724ffb7a8f57b02a373a2e816615b344f0bda8a7a80bee833a0R310

Reply:
From a closer look, I retract my statemnet. The only difference with handling any accelerator here would be that PrivateUse1 would be handled as well. But that mens that this would force the copy kernel to be done via copy_stub for privateuse1 as opposed to today where `_copy_from()` is being called and can be overwritten as any other dispatcher op.